packages/side-nav/src/SideNavLink.tsx

```diff
@@ -78,24 +82,12 @@ export const SideNavLink = ({
 			>
 				<Link href={href}>
 					{label}
-					{active && !children && (
```

Comment:
This is the bug. `&& !children`.  I think this came from an earlier implementation and should have been removed when I refactored.

Reply:
The side effect of applying this change is the group parent is also flagged as 'current page', which is the bug I am attempting to fix

Reply:
Hence why I have now added `isCurrentPage`

Reply:
That's weird, I was expecting that behaviour but didn't see it when I was testing. Must have been in a weird state.